### PR TITLE
Bump version for 7.9.3 release

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,14 +19,14 @@ ES-Hadoop 2.0.x and 2.1.x are compatible with Elasticsearch __1.X__ *only*
 
 ## Installation
 
-### Stable Release (currently `7.9.3`)
+### Stable Release (currently `7.9.4`)
```
Comment (jbaiera) [inline README.md:22]:
```suggestion
### Stable Release (currently `7.9.3`)
```

File: README.md [hunk 2]
```diff
@@ -19,14 +19,14 @@ ES-Hadoop 2.0.x and 2.1.x are compatible with Elasticsearch __1.X__ *only*
...
 Available through any Maven-compatible tool:
 
 ```xml
 <dependency>
   <groupId>org.elasticsearch</groupId>
   <artifactId>elasticsearch-hadoop</artifactId>
-  <version>7.9.3</version>
+  <version>7.9.4</version>
```
Comment (jbaiera) [inline README.md:29]:
```suggestion
  <version>7.9.3</version>
```

**Review by jbaiera — APPROVED** [review]
Thanks, LGTM, though it seems that at some point the current version on the README advanced a release number too soon. Left some suggest entries so it's a quick fix.

**Comment by matriv** [comment]
Thx @jbaiera! seemed odd and was going to double check.

**Pull request merged**